### ISSUE TITLE
Connection Issue

### ISSUE THREAD

**Issue by caljos13** [joffnerd/botw-trainer#121]
No matter what i do, Use DNS to block Nintendo servers, Use tcp gecko, use even mocha CFW, has the right ip in, no firewall or antivirus activated, I still get the error "Failed to connect" anyway to permanently squash this to the ground? like legit nothing i do works. Thanks

**Comment by xNeptune** [comment]
Your DNS, and mocha CFW won't affect the trainer's inability to connect to you. If you have any sort of antivirus make sure it's completely disabled and from the sounds of it you clearly have the correct IP address. Check your internet connection for your pc. Make sure that both the Wii U and the PC are using the same connection, your best bet is to have them both on the same wifi connection rather than ethernet.

**Comment by caljos13** [comment]
Interesting so yea Antivirus is off, and I do have the right IP, it tries to connect to it.  I unstalled hamachi, and disabled my firewall completely for that trainer. Just says on the error board in the trainer "Failed to connect"  I got it to work once, using loadiine but it just doesnt seem to do the trick, Should i use w4jml instead?

**Comment by GoodP** [comment]
It happens to me before. But then I realized that I manually entered IP address on wiiu that's what the gecko detecting. Then I found out that, My wii u aren't connected to my internet/ wifi router. haha

**Comment by caljos13** [comment]
My wii u ip is correct. Ive used 3 different ways, including going into the router itself to guarantee its the right ip

**Comment by joffnerd** [comment]
Have you done  everything the Readme says to do?

**Comment by caljos13** [comment]
Yes I used your specific instructions many times, I dont know what it is, It just gives me the error "Cant Connect" It connected once for about 5 minutes than disconnected after trying to change the speed of link.  But that was a long time ago, sorry im bothering you guys.

**Comment by caljos13** [comment]
Ok update, I put in the disc version of the game *USB* still attached, I go and connect it connects.  But if i try to do any items codes its an instand disconnect, any reason why?

**Comment by Megabyte918** [comment]
@caljos13 Are you by chance trying to use JGecko U/TCP Gecko and the Trainer at the same time? I've had this same error up until yesterday. It turns out you can only connect one of them to your Wii U at a time because they must all use the same port to connect.
TL;DR: If you are trying to run the Trainer and have either JGecko U or TCP Gecko running and connected to the same port, It wont work. You must disconnect the other.

**Comment by caljos13** [comment]
No, The only thing thats running is the trainer and my game, The only logical explanation i can come up with is that my computer connects via ethernet not wireless.  But when i tested it out on another computer that connects via wireless, it still doesnt work.

**Comment by slmenz90** [comment]
Yeah I'm having the same issue except I don't get an error message - the trainer appears to connect but when you go to the code section it's all blank 

Have used different SD cards, updated everything and tried it on a different wifi network too - no cigar :/
